supervisor/api/supervisor.py

```diff
@@ -143,6 +146,9 @@ async def options(self, request: web.Request) -> None:
         if ATTR_LOGGING in body:
             self.sys_config.logging = body[ATTR_LOGGING]
 
+        if ATTR_AUTO_UPDATE in body:
+            self.sys_updater.auto_update = body[ATTR_AUTO_UPDATE]
```

Comment:
If this changes from false to true we should imidiatly check for updates and only store the value to `self.sys_updater.auto_update` if we were able to check

Reply:
That is correctly handled today. We can force an updater fetch, but that is only a delta of less than 1h. I don't see an issue or abuse use case for this delta in all workflows. I guess let's fix issues in the flow once they going to be an issue and not over complicate it on the beginning for an feature that you basic let you at your own if you use it

Reply:
This option only controls whether we do updates of supervisor for them automatically. Setting it to `true` does not unlock any other updates. If supervisor isn't up to date then nothing else can be updated regardless of the value of this field.

And if people are blocking our ability to access the version file then there's not a whole lot we can do about that. And I don't think we should try to do much about it to be honest. Systems with an out of date supervisor are not supported. If a user blocking the version file submits an issue we'll know the system is unsupported because the supervisor version will be out of date, even if they've managed to dodge the unsupported flag.